prow/hook/server_test.go

```diff
@@ -17,6 +17,10 @@ limitations under the License.
 package hook
 
 import (
+	"bytes"
+	"github.com/google/go-cmp/cmp"
```

Comment:
Please put them in the next import group, so that we have one that only has stdlib imports

Reply:
Done (d19843cfdf), sorry about it. I assumed my IDE did that and haven't looked.